stellar/docs/link-token.md

```diff
@@ -0,0 +1,230 @@
+# Link Token Documentation
+
+## Overview
+
+This document describes the process for linking existing tokens across different chains using the Interchain Token Service (ITS) Hub.
+
+The token linking feature enables:
+
+- Linking existing tokens across connected Amplifier chains via ITS Hub
+- Supporting tokens with different decimal precisions through automatic scaling
+
+## How It Works
+
+The token linking process involves two key message types that ITS Hub uses to coordinate the linking:
+
+### Token Metadata Registration
+
+Before linking tokens, ITS Hub needs to know about each token's details (address and decimals) on both chains.
+
+### Token Linking
+
+Once metadata is registered, you can link tokens by specifying which tokens on different chains should be connected.
+
+### Decimal Scaling
+
+ITS Hub automatically handles decimal scaling between linked tokens:
+
+- **Stellar Token**: 7 decimals (1 USDC = 10,000,000 units)
+- **EVM Token**: 18 decimals (1 USDC = 1,000,000,000,000,000,000 units)
+- **Scaling Factor**: 10^(18-7) = 10^11
+
+When transferring 1 USDC from Stellar to EVM:
+
+- Stellar: 10,000,000 units locked/burned
+- EVM: 1,000,000,000,000,000,000 units unlocked/minted
+
+## Link Token Flow
+
+```
+1. User controls Token A (Stellar) and Token B (EVM)
+2. User → ITS Stellar: registerTokenMetadata(Token A) → ITS Hub
+3. User → ITS EVM: registerTokenMetadata(Token B) → ITS Hub
+4. User → ITS Stellar: registerCustomToken() → Deploys Token Manager A
+5. User → ITS Stellar: linkToken() → ITS Hub
+6. ITS Hub: Calculates scaling factor from stored decimals
+7. ITS Hub → ITS EVM: Deploy Token Manager B
```

Comment:
What ITS hub does is a design detail, whereas the flow should cover what the user needs to do to link the token